card/lib/card/cache.rb

```diff
@@ -40,6 +40,11 @@ def renew
         reset_local
       end
 
+      def prefix_root
```

Comment:
This was done to help with some issues where Cardio.config wasn't initialized when this was loaded.  I'm not sure what use case it was, but it's just a lazy load so it doesn't have to be initialize until it is used vs. when defined.